crates/forge_tracker/src/dispatch.rs

```diff
@@ -91,6 +98,7 @@ impl Tracker {
                 email: email.clone(),
                 model: self.model.lock().await.clone(),
                 conversation: self.conversation().await,
+                login: self.login.lock().await.clone(),
```

Comment:
Login should be a person property.

https://posthog.com/docs/product-analytics/person-properties

Reply:
You might want to try using - https://posthog.com/docs/api/capture#identify